src/_tutorials/language/futures.md

```diff
@@ -175,45 +174,59 @@ program prints.  If `gatherNewsReports()` takes a little while to complete its
 work, no great harm is done: the user gets to read other things before the
 daily news digest is printed.
 
+Note the return types. The return type of `gatherNewsReports()` is
+`Future<String>`, which means that it returns a Future that completes with
+a String value. The `printDailyNewsDigest()` function, which doesn't
+return a value, has the return type `Future<void>`.
+
 The following diagram shows the flow of execution through the code.  Each
 number corresponds to a step below.
 
 <img src="../images/async-await.png"
-     alt="diagram showing flow of control through the main() and printDailyNewsDigest functions" />
+     alt="diagram showing flow of control through the main() and printDailyNewsDigest functions">
 
 1. The app begins executing.
-1. The main function calls `printDailyNewsDigest()`, which (because it's
-   marked `async`), immediately returns a Future, _before any code is
-   executed_.
+1. The `main()` function calls the async function `printDailyNewsDigest()`,
+   which begins executing synchronously.
```

Comment:
so technically yes, this part is true, because printDailyNewsDigest() is not awaited. But if that's important to cover, this now needs to be made clear!

Reply:
Even if it's awaited, it begins executing synchronously! (See https://dartpad.dartlang.org/b62c036c308fd018ff37da5d77670140.)

Reply:
Oh man, I am confused

Reply:
Yes, this was the change for Dart 2: async functions begin synchronously.